backend/tests/functional.containerfile

```diff
@@ -0,0 +1,27 @@
+FROM quay.io/centos/centos:stream9
+
+ENV PATH=/root/.local/bin:$PATH \
+    LANG=C.UTF-8 \
+    PYTHONPATH=/backend
+
+RUN mkdir -p /backend/tests
+
+COPY pyproject.toml /backend
+COPY poetry.lock /backend
+
+WORKDIR /backend
+
+RUN dnf install -y pip gcc python3-devel gcc-c++
+
+RUN pip install --user poetry && \
+    poetry self add poetry-plugin-export && \
+    poetry export --without-hashes -f requirements.txt -o requirements.txt && \
+    pip install -U typing-extensions && \
+    pip install --user dash && \
+    pip install --no-cache-dir -r requirements.txt
+
+ADD tests/ /backend/tests/    
+
+ENV SERVER=http://localhost:8000
+
+ENTRYPOINT ["pytest", "-s", "tests/functional"]
```

Comment:
Interesting ... I guess this doesn't matter since you're overriding it; and if you changed it, I'd need to restore it later ... but then, I'll need to "fix" the `--entrypoint` anyway, so maybe it makes more sense just to package your seed script in this container when you build it, and set that as the entrypoint here?

Then again, I wonder if a separate container image for seeding Opensearch instance(s) would be better -- since (as you're doing in your script) we'll want the seed to run inline and first rather than asynchronously, whereas the functional tests can be asynchronous along with the Cypress tests later. I don't think there's a way of untangling all these with this double-duty, and it's easy enough to build another container image instead of using this!

Reply:
what does this mean you want to do?